android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java

```diff
@@ -626,7 +627,10 @@ private void handleUri(String uri) {
   }
 
   public void emitUpdatesEvent(JSONObject params) {
-    KernelProvider.getInstance().addEventForExperience(mManifestUrl, new KernelConstants.ExperienceEvent(AppLoader.UPDATES_EVENT_NAME, params.toString()));
+    String eventName = ABIVersion.toNumber("39.0.0") >= ABIVersion.toNumber(mSDKVersion)
```

Comment:
This version check logic looks possibly inverted -- wouldn't we want to continue to emit the old `AppLoader.UPDATES_EVENT_NAME` for older SDK versions?

```suggestion
    String eventName = ABIVersion.toNumber("39.0.0") <= ABIVersion.toNumber(mSDKVersion)
```

Reply:
I suppose this doesn't really matter after https://github.com/expo/expo/pull/9717/.

Reply:
Good catch, thanks 😅